src/bitcoin/Bitcoin.mo

```diff
@@ -119,7 +120,7 @@ module {
             // Create Script Sig which looks like:
             // ScriptSig = <Signature> <Public Key>.
             [
-              #data (Blob.toArray(ecdsaProxy.sign(Blob.fromArray(sighash)))),
+              #data (Blob.toArray(ecdsaProxy.sign(Blob.fromArray(sighash), []))),
               #data (PublicKey.toSec1(ecdsaProxy.publicKey(), false).0)
```

Comment:
> // `ecdsaProxy` is a temporary interface for ecdsa signing used for testing.

In my opinion, we should use in a permanent way the `EcdsaProxy`.
If we add `derivation_path` and `compressed` parameters to `signTransaction`, [my Motoko Basic Example](https://github.com/dfinity/examples/pull/281/files#diff-5b452b3947288002cd0b0bf245010061a0f4ad295a469135b7d388cd2f0c2d2aR152-R200) could likewise directly use this function.